src/dataconnect/fileUtils.ts

```diff
@@ -106,3 +106,30 @@ export async function pickService(
   }
   return serviceInfo;
 }
+
+// case insensitive indicators for supported app platforms
+const WEB_INDICATORS = ["package.json", "package-lock.json", "node_modules"];
+const IOS_INDICATORS = ["info.plist", "project.pbxproj", "podfile", "package.swift"];
+const IOS_FOLDER_INDICATORS = [".xcworkspace", ".xcodeproj"];
+const ANDROID_INDICATORS = ["androidmanifest.xml", "build.gradle"];
+
+// given a directory, determine the platform type
+export async function getPlatformFromFolder(dirPath: string) {
+  // check for ios folder indicator
+  if (IOS_FOLDER_INDICATORS.some((indicator) => path.basename(dirPath).endsWith(indicator))) {
+    return Platform.IOS;
```

Comment:
Is this comparing the app folder path or a file / folder inside the app folder. (related to my other comment)